jadx-gui/src/main/java/jadx/gui/settings/JadxProject.java

```diff
@@ -39,19 +40,23 @@
 			.create();
 
 	private transient MainWindow mainWindow;
-	private transient JadxSettings settings;
 
 	private transient String name = "New Project";
 	private transient Path projectPath;
 
 	private transient boolean initial = true;
 	private transient boolean saved;
 
+	@SerializedName("files")
```

Comment:
I think these annotations not needed if name equals field name.

Reply:
I know, but it is more correct to do it for json serialization.  In general, it should be one model class that describes the structure of the project/settings...